scripts/generate-icon-exports.js

```diff
@@ -1,49 +1,139 @@
 const fs = require('fs');
+const util = require('util');
 const path = require('path');
 const glob = require('glob');
+const del = require('del');
+const svgr = require('@svgr/core').default;
+
+// Convert fs.readFile / writeFile into Promise version of same
```

Comment:
Q: is there a reason to not use the `sync` version of the functions?

Reply:
no not really, I just thought it was better practice not to. I guess for a script it doesn't matter?

Reply:
Yeah a script usually run synchronously. At least if feels a bit weird to force synchronous function into async functions.